Dockerfile

```diff
@@ -2,10 +2,11 @@ FROM golang:1.19-alpine3.16 AS spicedb-builder
 WORKDIR /go/src/app
 RUN apk update && apk add --no-cache git
 COPY . .
-RUN go build -v ./cmd/spicedb/
+RUN echo 'hosts: files dns' > /tmp/nsswitch.conf
+RUN CGO_ENABLED=0 go build -v ./cmd/spicedb/
 
-FROM alpine:3.16.2
-RUN [ ! -e /etc/nsswitch.conf ] && echo 'hosts: files dns' > /etc/nsswitch.conf
-COPY --from=ghcr.io/grpc-ecosystem/grpc-health-probe:v0.4.6 /ko-app/grpc-health-probe /usr/local/bin/grpc_health_probe
+FROM distroless.dev/static
+COPY --from=spicedb-builder /tmp/nsswitch.conf /etc/nsswitch.conf
```

Comment:
distroless comes with an nsswitch that checks hosts files first: https://github.com/GoogleContainerTools/distroless/pull/180

Reply:
@ecordell I checked the image contents and it did not contain any nsswitch.conf file, unless I'm missing something? I used `dive` to instrospect the image

Reply:
Ah, I didn't realize the distroless.dev images were so different, thanks for noticing that!

You're right, we should have an nsswitch in both images. Could you add that line to the Dockerfile.release file?

It would be great to not have to worry about this for any of our images, so I filed https://github.com/distroless/meta/issues/7 to see if that's a change we could contribute the distroless images.

Reply:
addressed in https://github.com/authzed/spicedb/pull/797/commits/3aad4aa26cc8ca4334d660a8c0f4224565e183af